node/pkg/watchers/evm/verify_chain_config/verify.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+// This is a tool that queries the RPCs to verify that the `EvmChainID` values in the chain config maps are correct as
+// compared to the specified well-known public RPC endpoint. This tool should be run whenever either of the chain config maps are updated.
+
+// Usage: go run verify.go [--env `env``]
+//        Where `env` may be "mainnet", "testnet" or "both" where the default is "both".
+
```

Comment:
Should this be added as part of GH ci action?

Reply:
I guess I view this as similar to the governor prices tables, when someone regenerates it they need to verify their changes before they commit.

But I suppose we could add a GH action in a follow-on PR if we think it's worthwhile.